src/construct_realm.js

```diff
@@ -17,9 +17,19 @@ import * as evaluators from "./evaluators/index.js";
 import * as partialEvaluators from "./partial-evaluators/index.js";
 import { NewGlobalEnvironment } from "./methods/index.js";
 import { ObjectValue } from "./values/index.js";
+import { Debugger } from "./debugger/Debugger.js";
+const fs = require("fs");
 
 export default function(opts: RealmOptions = {}): Realm {
   let r = new Realm(opts);
+  if (r.enableDebugger) {
+    let dbgFileContents = fs.readFileSync("./src/debugger/.sessionlogs/proxy2debugger.txt", "utf8");
```

Comment:
Do not assume that prepack is being run from a particular directory. Rather parameterize this and supply it from your test runner.

Reply:
Changed